src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/INotifyPropertyChangedExtension.shared.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace Xamarin.CommunityToolkit.ObjectModel.Extensions
 {
-	public static class INotifyPropertyChangedEx
+	public static class INotifyPropertyChangedExtension
 	{
-		public static void WeakSubscribe<T>(this INotifyPropertyChanged target, T subscriber, Action<T, object, EventArgs> action)
+		public static void WeakSubscribe<T>(this INotifyPropertyChanged target, T subscriber, Action<T, object, PropertyChangedEventArgs> action)
```

Comment:
Looks like that this change has nothing with the current ticket. Can you revert this? If you want to change anything else I suggest opening a new issue and a new PR. That way is easier for us to track changes.

Reply:
Thanks for explaining. No PR is too small. Got it. Didn't want to create whole new PR just for this change (bad habit of mine).

Reply:
Thanks <3

Reply:
Could you please re-approve the PR?